CRM/Core/Resources/Common.php

```diff
@@ -204,6 +204,10 @@ protected static function coreResourceList($region) {
       "js/wysiwyg/crm.wysiwyg.js",
     ];
 
+    $basePage = $settings->get('wpBasePage');
```

Comment:
This makes me squeamish - it smells like coupling between Civi and a particular CMS brand, which we try to avoid.
One thing that would make it smell better was if the setting were named "CMSBasePage".
Or maybe there's an even better solution, IDK.

Reply:
I think it's only Wordpress that actually needs/provides the setting. Drupal/Joomla have no such concept and it's *always* `civicrm`.

Reply:
The setting itself exists on all CMSs